brokers/theia-plugin-broker/broker/broker_test.go

```diff
@@ -0,0 +1,384 @@
+//
+// Copyright (c) 2012-2018 Red Hat, Inc.
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Contributors:
+//   Red Hat, Inc. - initial API and implementation
+//
+
+package broker
+
+import (
+	"errors"
+	"fmt"
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+
+	tests "github.com/eclipse/che-plugin-broker/brokers_test"
+	cmock "github.com/eclipse/che-plugin-broker/common/mocks"
+	fmock "github.com/eclipse/che-plugin-broker/files/mocks"
+	"github.com/eclipse/che-plugin-broker/model"
+	"github.com/eclipse/che-plugin-broker/storage"
+)
+
+var (
+	bMock      = &cmock.Broker{}
+	uMock      = &fmock.IoUtil{}
+	mockBroker = &TheiaPluginBroker{
+		bMock,
+		uMock,
+		storage.New(),
+	}
+)
+
+func Test_process_remote_plugin(t *testing.T) {
```

Comment:
AFAIU According to https://golang.org/pkg/testing/ it should be named `TestProcessRemotePlugin`. Why did you choose naming with underscores?

Reply:
I'll fix it